src/components/DateInput/Day.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import dayjs, { Dayjs } from 'dayjs';
+import { noop } from '../../utils/helpers';
+
+export interface DayProps {
+  day?: number;
+  month: number;
+  year: number;
+  daySelected?: Dayjs;
+  onDaySelect?: (date: Dayjs) => void;
+}
+
+const Cell = styled.div`
+  display: table-cell;
+  padding: ${({ theme }) => `${theme.space[2]}px`};
+  vertical-align: middle;
+  text-align: center;
+  cursor: pointer;
+  border-radius: 50%;
+  &[aria-selected='true'],
+  &[aria-selected='true']:hover {
+    background-color: ${({ theme }) => theme.colors['blue-400']};
+  }
+  &:hover {
+    background-color: ${({ theme }) => theme.colors['navyblue-300']};
+  }
+`;
+
+const DisabledCell = styled(Cell)`
```

Comment:
with the `<Box>` implementation above, `DisabledCell` doesn't even need to be a component. It could be `<Cell disabled />`.

Styleds for `disabled` get automatically added from `GlobalStyles.tsx` which detects the presence of the selector `&[disabled], &[aria-disabled], &[data-disabled]`